core/src/main/java/org/apache/struts2/dispatcher/Dispatcher.java

```diff
@@ -986,10 +986,12 @@ private void applyEncoding(HttpServletResponse response, String encoding) {
     public HttpServletRequest wrapRequest(HttpServletRequest request) throws IOException {
         // don't wrap more than once
         if (request instanceof StrutsRequestWrapper) {
+            LOG.debug("Request already wrapped with {}", StrutsRequestWrapper.class.getSimpleName());
```

Comment:
```suggestion
            LOG.debug("Request already wrapped with: {}", StrutsRequestWrapper.class.getSimpleName());
```

to be consistent with other logs from this PR